src/main/java/com/cb/services/dbService/dbServiceImpl/CharacterDBServiceImpl.java

```diff
@@ -25,25 +25,25 @@ public void setTemplate(JdbcTemplate template) {
     @Override
     public List<CharacterDAL> getCharacters() {
 
-        return template.query("SELECT * FROM main.character", new BeanPropertyRowMapper(CharacterDAL.class));
+        return template.query("SELECT * FROM main7.character", new BeanPropertyRowMapper(CharacterDAL.class));
 
     }
 
     public int getCharacterId(FighterBL fighterBL) {
 
-       return template.queryForObject("SELECT Id from main.character where Charname = '" + fighterBL.getMember() + "'", Integer.class);
+       return template.queryForObject("SELECT Id from main7.character where Charname = '" + fighterBL.getMember() + "'", Integer.class);
 
     }
 
     public String getCharacterName(int charId) {
 
-        return template.queryForObject("SELECT Charname from main.character where Id = '" + charId + "'", String.class);
+        return template.queryForObject("SELECT Charname from main7.character where Id = '" + charId + "'", String.class);
 
     }
 
     public String getImageReference(int charId) {
 
-        return template.queryForObject("SELECT imageReference from main.character where Id = '" + charId + "'", String.class);
+        return template.queryForObject("SELECT imageReference from main7.character where Id = '" + charId + "'", String.class);
```

Comment:
Now it will not work for everyone and you are pushing not working code for everyone.